modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/form/CustomSelect/CustomSelect.scss

```
@@ -0,0 +1,21 @@
+.custom-select-trigger {
+	background-color: #f1f2f5;
+	color: #272833;
+	cursor: pointer;
+
+	&:focus {
+		background-color: #f0f5ff;
+		border-color: #80acff;
```

use variables instead of hard code color codes:
```suggestion
		background-color: $primary-l3;
		border-color: $primary-l1;
```
[more info](https://liferay.atlassian.net/wiki/spaces/~294232222/pages/1842741280/Code+Review+Common+Mistakes#Color-Variables)